### PR TITLE
front: drop hardcoded scenario header height

### PR DESCRIPTION
**Pull request by emersion**
The scenario header size is hardcoded to 8rem. Replace these rules with a flexbox-based layout.

### CODE REVIEW

File: front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx [hunk 1]
```diff
@@ -158,110 +158,110 @@ const ScenarioV2 = () => {
       <main className="mastcontainer mastcontainer-no-mastnav">
         <div className="scenario">
           <div className="row scenario-container">
-            <div className={collapsedTimetable ? 'd-none' : 'col-hdp-3 col-xl-4 col-lg-5 col-md-6'}>
-              <div className="scenario-sidemenu">
```
Comment (emersion) [inline front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx:162]: Note for reviewers: this big diff is caused by merging together these two `<div>` elements. There is no change below apart from removing one indent level.

Comment (woshilapin) [reply front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx:162]: For those who don't know, in the ⚙️ above in the "Files changed" tab, the following option "Hide whitespace" might help

image

**Review by Uriel-Sautron — APPROVED** [review]
LGTM thank you!!! Tested

**Review by theocrsb — APPROVED** [review]
LGTM

**Review by clarani — APPROVED** [review]
LGTM, tested :white_check_mark:

**Pull request merged**